### PR TITLE
Add empty and null scrubbers

### LINKED ISSUE #19: How to null a given field
Hi @costela 

I was wondering... how can I null a given field? I looked at faker but there as well not information about what to do.

Any ideas about that?

Thx!  
Ronny

### PR DESCRIPTION
**Pull request by costela**
fixes #19

### CODE REVIEW

**Comment by GitRon** [comment]
👍

File: django_scrubber/scrubbers.py [hunk 1]
```diff
@@ -132,9 +139,9 @@ def _initialize_data(self):
 
         provider_args_str = ', '.join(str(i) for i in self.provider_args)
         provider_kwargs_str = ', '.join(str(i) for i in self.provider_kwargs)
-        logger.info('Initializing fake scrub data for provider %s(%s, %s)' %
-            (self.provider, provider_args_str, provider_kwargs_str)
-        )
+        logger.info('Initializing fake scrub data for provider %s(%s, %s)' % (
```
Comment (MounirMesselmeni) [inline django_scrubber/scrubbers.py]: Maybe use logger string formatting which may not be interpreted if the logger is disabled
```suggestion
        logger.info('Initializing fake scrub data for provider %s(%s, %s)',
```

File: django_scrubber/scrubbers.py [hunk 2]
```diff
@@ -132,9 +139,9 @@ def _initialize_data(self):
...
+            self.provider, provider_args_str, provider_kwargs_str
+        ))
```
Comment (MounirMesselmeni) [inline django_scrubber/scrubbers.py]:
```suggestion
        )
```

File: django_scrubber/scrubbers.py (hunk 1 above)
Comment (costela) [reply django_scrubber/scrubbers.py]: good point. Done!

**Pull request merged**